code/modules/surgery/plastic_surgery.dm

```diff
@@ -30,12 +30,42 @@
 		user.visible_message("[user] successfully restores [target]'s appearance!", "<span class='notice'>You successfully restore [target]'s appearance.</span>")
 	else
 		var/list/names = list()
+		var/list_size = 10
+		var/obj/item/card/id/ID
+
+		//IDs in hand
+		if(istype(user, /mob/living/carbon/human)) //Only 'humans' can hold ID cards
+			var/mob/living/carbon/human/H = user
+			ID = H.get_id_from_hands()
+			if(ID)
+				names += ID.registered_name
+				list_size -- //To stop list bloat
```

Comment:
spacing.
`++` and `--` always go attached to the variable they belong to. So `i++` or `i--`

Reply:
So just `list_size--`?

Reply:
Yep yep